dist/index.html

```diff
@@ -25,6 +25,10 @@
 
             Hoot.api.getServicesVersionInfo()
                 .then( () => {
+                    // apply cached hash
+                    window.location.hash = localStorage.getItem('hootQueryString');
```

Comment:
We don't always want to do this because if there's not a query string in local storage (such as when already logged in) this will whack the existing hash that has things like map location/zoom.